packages/@aws-cdk/aws-neptune-alpha/lib/instance.ts

```diff
@@ -403,6 +403,13 @@ export interface DatabaseInstanceProps {
    * @default RemovalPolicy.Retain
    */
   readonly removalPolicy?: cdk.RemovalPolicy;
+
+  /**
+   * Indicates that minor version patches are applied automatically.
+   *
+   * @default - same as the cluster's autoMinorVersionUpgrade setting
```

Comment:
the default in CDK is undefined. Do not refer to the default value in the service.

Reply:
So, does this mean that it is preferable to specify @default undefined here?

Reply:
yes

Reply:
The DESIGN_GUIDELINES says that it is better to avoid `@default default`.

```md
don't write @default undefined, describe the behavior that happens if the property is not supplied.
```

<https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#defaults>

Therefore, I think it would be better to describe how the current behavior works. What do you think?

Reply:
but IMO, we should not mention the CFN default behaviour, since we will not update this value if CFN decided to change that default value.

Reply:
I've updated my description to `@default undefined`.